main.go

```diff
@@ -92,9 +94,15 @@ func main() {
 
 	klog.Info("Creating the cable engine")
 
-	localCluster, err := util.GetLocalCluster(submSpec)
-	if err != nil {
-		klog.Fatalf("Error creating local cluster object from %#v: %v", submSpec, err)
+	localCluster := types.SubmarinerCluster{
+		ID: submSpec.ClusterID,
+		Spec: subv1.ClusterSpec{
+			ClusterID:   submSpec.ClusterID,
+			ColorCodes:  submSpec.ColorCodes,
+			ServiceCIDR: submSpec.ServiceCidr,
+			ClusterCIDR: submSpec.ClusterCidr,
+			GlobalCIDR:  submSpec.GlobalCidr,
+		},
```

Comment:
Just for having a shorter main, I'd keep it on a function, may be `localCluster := GetClusterFromSpec(submSpec)`

of course, without return of error which made no sense.

Reply:
Done